### PR TITLE
Implement storing and displaying of pictures for questions

### LINKED ISSUE #130: Implement storing and displaying of pictures for questions

### PR DESCRIPTION
**Pull request by fantablack**
Closes #130 

**Summary**
* Questions currently only consist of string data, which may not be sufficient as some questions require image examples to help the user better understand the question.
* Update Question Bank and Question Schema to support base64 encoded image data.
* Implement rendering of image data if it exists in `RoomPage.js` and `QuestionDisplay.js`.
* To test with a question that has an image: 
   * In `question-controller.js`, update line 29 to `await _findQuestionRandomly(level, 'Rotting Oranges');` to force load that question. 
   * When selecting question difficulty in the frontend, select Medium Difficulty, as Rotting Oranges is a Medium question.
   * Observe the image loaded in the question display area.

**Example of question with image (Updated)**
withImage

**Example of question without image**
withoutImage

### CODE REVIEW

File: question-service/model/question-orm.js [hunk 1]
```diff
@@ -41,11 +41,11 @@ export async function ormUpdateQuestion(Question, changes) {
     }
 }
 
-export async function ormFindQuestionRandomly(difficulty, title=null) {
+export async function ormFindQuestionRandomly(difficulty, title) {
     try {
         const params = {};
         if (title) {
-            params['QuestionTitle'] = {$ne : title};
+            params['QuestionTitle'] = title;
```
Comment (ureshiiYing) [inline question-service/model/question-orm.js:48]: Why was this changed? This needs to be in place in order to ensure that when generating new questions, the current question does not appear again. While I understand that it might be for testing purposes, it should not be left in the PR.

Comment (fantablack) [reply question-service/model/question-orm.js:48]: @ureshiiYing  I see! Thanks for pointing it out. I misunderstood its purpose and removed it as it allowed my feature to work before & after testing. I will add it back.

**Review by ureshiiYing — APPROVED** [review]
Tested, works, LGTM!
